Marlin/pins_ULTIMAIN_2.h

```diff
@@ -110,3 +118,18 @@
 #define BTN_EN1            40
 #define BTN_EN2            41
 #define BTN_ENC            19
+
+// Rev B additions
+
+//
+// M3, M4 & M5 (spindle control) support
+//
+#if ENABLED(SPINDLE_LASER_ENABLE)   // use the LED_PIN for spindle speed control or case light
+  #undef  LED_PIN
+  #define SPINDLE_DIR_PIN    16
+  #define SPINDLE_LASER_ENABLE_PIN 17  // should have a pull up
+  #define SPINDLE_SPEED_LASER_POWER_PIN   8  // must have a pull up
+#else                        // Case light
```

Comment:
Ok, so I'd like to find a way to get rid of lines like these in the pins files. We've been trying to make the pins files as agnostic as possible about what features are enabled, so they simply supply the recommended (or required) pins for each feature. We shouldn't be checking the presence of a pin by itself when enabling code.

We've tried some different approaches to this, but generally the idea is to make sure we don't need a lot of `#if` clauses in the pins files that relate to features.